.gitignore

```diff
@@ -1,73 +1,10 @@
-### Windows ###
-
-thumbs.db
-*.db
-
-### Java ###
-
-*.class
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.war
-*.ear
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-
-### Eclipse ###
-
-*.pydevproject
-.metadata
-.gradle
-bin/
-tmp/
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.settings/
-.loadpath
-
-# Eclipse Core
-.project
-
-# External tool builders
-.externalToolBuilders/
-
-# Locally stored "Eclipse launch configurations"
-*.launch
-
-# CDT-specific
-.cproject
-
-# JDT-specific (Eclipse Java Development Tools)
-.classpath
-
-# Java annotation processor (APT)
-.factorypath
-
-# PDT-specific
-.buildpath
-
-# sbteclipse plugin
-.target
-
-# TeXlipse plugin
-.texlipse
-
-### Intellij IDEA ###
-
-*.iml
 *.ipr
 *.iws
-
-.idea/
+ .idea/*
+*.log
 .idea_modules/
-
-/out/
-eclipse/
-*.bat
+*.xml
+*.log
+*.iml
+*.log
```

Comment:
Do we need *.log twice?

Reply:
Any reason we are removing the comments that make it easier to understand the gitignore?